.github/workflows/release.yml

```diff
@@ -108,15 +119,13 @@ jobs:
           platforms: all
 
       - name: Install cibuildwheel
-        run: python3 -m pip install cibuildwheel==2.4.0
+        run: python3 -m pip install cibuildwheel==2.16.5
 
       - name: Build wheels
         run: python3 -m cibuildwheel --output-dir wheelhouse
         env:
-          CIBW_SKIP: "pp* *-win32"
-          CIBW_ARCHS_WINDOWS: "auto"
-          CIBW_ARCHS_LINUX: "auto aarch64"
-          CIBW_ARCHS_MACOS: "x86_64"
+          CIBW_SKIP: "pp* *-win32 cp36-* cp37-* cp38-*"
```

Comment:
Prefer explicit including instead of accepting the `cibuildwheel` defaults and skipping some. That way we _know_ that we are only building for arch/python version combinations that we have tested for and it won't change with `cibuildwheel`.

Reply:
Please also add a comment about what these apparently arcane flags mean for the next dev that has to change this.

Reply:
Added a commit to
* uses specific arch names instead of alias "auto",
* uses specific python versions 3.9 to 3.11
* Adds a link to `cibuildwheel` document

I tried to add python 3.12 but the build failed due to a dependency issue. I'll open a new issue to track the 3.12 build issue.

Reply:
Opened a new issue for adding 3.12 to the build workflow - https://github.com/amazon-ion/ion-python/issues/339